orangecontrib/text/widgets/owconcordance.py

```diff
@@ -0,0 +1,218 @@
+from collections import Counter
+from typing import Optional
+
+from AnyQt.QtCore import Qt, QAbstractTableModel, QSize
+from AnyQt.QtWidgets import QSizePolicy, QApplication, QTableView
+from AnyQt.QtGui import QBrush, QColor
+
+
+from Orange.data import Table
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.preprocess import Preprocessor, WordPunctTokenizer
+
+
+class IO:
+    DATA = 'Corpus'
```

Comment:
Copied from Niko's corpusviewer.py. I won't pretend I know what it's doing there, I though it was necessary to assure the input is Corpus, otherwise the widget won't work.